agi_plz_ortschaften_pub/build.gradle

```diff
@@ -53,15 +66,42 @@ task dbImport(type: Ili2pgImport, dependsOn: 'unzipData'){
     strokeArcs = true
 }
 
+task convertDataEdit(type: SqlExecutor, dependsOn: 'dbImportSogis'){
+    description = "Convert plz_ortschaften-Data to Edit Schema"
+    doLast {
+        println "plz_ortschaften-Data are converted."
+    }
+	database = [dbUriEdit, dbUserEdit, dbPwdEdit]
+    sqlFiles = [
+            'delete_ortschaft.sql',
+            'delete_postleitzahl.sql',
+            'insert_ortschaft.sql',
+            'insert_postleitzahl.sql',
+	]
+}
 
-task convertData(type: Db2Db, dependsOn: 'dbImport'){
-    description = "Convert plz_ortschaften-Data"
+task convertDataSogis(type: SqlExecutor, dependsOn: 'convertDataEdit'){
```

Comment:
Innerhalb der sogis-DB braucht es keinen Umbau ins Publikationsmodell, denn in der sogis-DB ist im Moment nur das Originalmodell, also braucht es dort das Publikationsmodell nicht. Diesen Task kannst du also löschen.